app/schemas/users.py

```
@@ -0,0 +1,110 @@
+# ===== Requests =====
+class UserSignupRequest(BaseModel):
+    email: str
+    password: str = Field(min_length= 2, max_length=20)
+    nickname: str
+    profile_image: str | None = None
+
+    @field_validator("password")
+    @classmethod
+    def check_password(cls, v: str) -> str:
+        if not re.search(r"\d", v) and not re.search(r"[^\w\s]", v):
+            raise ValueError("비밀번호에는 숫자와 특수문자가 반드시 포함되어야 합니다.")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)